src/OrchardCore.Modules/OrchardCore.Contents/Liquid/ContentAnchorTag.cs

```diff
@@ -230,6 +214,13 @@ public async ValueTask<Completion> WriteToAsync(List<FilterArgument> argumentsLi
             tagBuilder.RenderEndTag().WriteTo(writer, (HtmlEncoder)encoder);
 
             return Completion.Normal;
+
+            async Task<ContentItemMetadata> PopulateAspectForContentItemMetadataAsync(ContentItem item)
+            {
+                contentItem = item;
```

Comment:
Now this contentItem copy is useless because it is not used in the context of this method.